ui/signed-out/src/commonMain/kotlin/social/androiddev/signedout/composables/SignInContent.kt

```diff
@@ -0,0 +1,104 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentHeight
+import androidx.compose.material.Icon
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TopAppBar
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import kotlinx.coroutines.delay
+import social.androiddev.common.composables.webview.SignInWebView
+import social.androiddev.signedout.navigation.SignInComponent
+
+@Composable
+fun SignInContent(
+    server: String,
+    component: SignInComponent,
+    modifier: Modifier = Modifier,
+) {
+
+    Column(
+        modifier = modifier
+    ) {
+        TopAppBar(
+            modifier = Modifier.fillMaxWidth(),
+            title = {
+                Text("Login")
+            },
+            navigationIcon = {
+                Icon(
+                    modifier = Modifier.clickable {
+                        component.onCloseClicked()
+                    },
+                    imageVector = Icons.Default.Close,
+                    contentDescription = "Close",
+                )
+            }
+        )
+
+        var errorVisibilityState by remember { mutableStateOf(false) }
+        AnimatedVisibility(errorVisibilityState) {
+            Box(
+                modifier = Modifier.fillMaxWidth()
+                    .wrapContentHeight()
+                    .align(Alignment.CenterHorizontally)
+                    .padding(16.dp)
+                    .background(color = MaterialTheme.colors.error.copy(alpha = 0.5F))
+                    .border(
+                        border = BorderStroke(
+                            width = 2.dp,
+                            color =
+                            MaterialTheme.colors.error
+                        ),
+                        shape = MaterialTheme.shapes.small
+                    )
+                    .padding(16.dp),
+                contentAlignment = Alignment.Center
+            ) {
+                Text(
+                    "Something is wrong!",
+                    color = MaterialTheme.colors.onError,
+                    style = MaterialTheme.typography.subtitle2
+                )
+            }
+
+            LaunchedEffect(errorVisibilityState) {
+                delay(3000)
+                errorVisibilityState = false
+            }
+        }
+        SignInWebView(
+            server = server,
+            onSignedIn = { component.onSignInSucceed() },
+            onFailed = { errorVisibilityState = true }
```

Comment:
Maybe we could display the error message from error if if `onFailed` delivers error

Reply:
For the moment I display a general error, son In future work, we will manage all the potential errors raised by the webview.

Reply:
Yes, we will improve this in the follow-up PR